docs/Guide.Expo.md

```diff
@@ -0,0 +1,44 @@
+---
+id: Guide.Expo
+title: Expo
+---
+
+## Usage with Expo
+
+- Install `detox` and `detox-expo-helpers` (yarn or npm)
+- Add `detox` configuration to [package.json](https://github.com/expo/with-detox-tests/blob/master/package.json#L21-L29):
+
+```es6
+"detox": {
+  "configurations": {
+    "ios.sim": {
+      "binaryPath": "bin/Exponent.app",
+      "type": "ios.simulator",
+      "name": "iPhone 7"
+    }
+  }
+}
+```
+
+- Download the Expo app from [Expo.io/tools](https://expo.io/tools). As of 03/19/18
+  - [iOS IPA 2.3.0](https://dpq5q02fu5f55.cloudfront.net/Exponent-2.3.0.tar.gz)
```

Comment:
Try being less specific here, we don't want to update this regularly. Just put a link to the download page (https://expo.io/tools)